src/components/Personalization/groupDecisions.js

```diff
@@ -104,8 +104,6 @@ const getGroupedDecisions = unprocessedDecisions => {
   };
 };
 
-export default {
-  groupDecisions(decisions) {
-    return getGroupedDecisions(decisions);
-  }
+export default decisions => {
+  return getGroupedDecisions(decisions);
 };
```

Comment:
Preferably rename `getGroupedDecisions` to `groupDecisions` and then just shorten this to `export default groupDecisions;`.

Reply:
ok, I will rename it. Thanks